metersink/api.py

```diff
@@ -20,29 +20,43 @@ def _init_logger():
 
 CONFIG = None
 
+def push_to_sinks(conf, data):
+    """puts received metering data to the configured billing sinks"""
+    # LOG.debug('%s', conf)
+    sinks = get_sinks(conf)
+    LOG.debug("pushing to sinks: %s", sinks)
+    for sink_type, sink_values in sinks.items():
+        if sink_type == "file":
+            for index, sink_name in enumerate(sinks["file"]["name"]):
+                # maybe we want to differ between events and polls here
+                # for now we put all incoming into the file
+                output_file(sink_name, data)
+        elif sink_type == "odoo":
+            odoo_handle(sinks['odoo'], conf, data)
```

Comment:
Two more things that I don't really understand.

- Why `enumerate`? The `index` is not used at all?
- Why use the itemgetter `sinks["file"]` and `sinks["odoo"]` -- this is just `sink_values` in both cases.

Reply:
the enumeration is necessary for configuration of multiple sinks of the same type. I put more examples into the settings template. maybe this explains it better. Also the unused value variables are given as a "mind-hook". for adding more features it could be necessary to use them in future soon.

Reply:
We are producing a PoC for people to learn from, and they will be baffled by complications that will only make sense in the future.

```suggestion
    for sink_type, sink_values in sinks.items():
        if sink_type == "file":
            for sink_name in sink_values["name"]:
                # maybe we want to differ between events and polls here
                # for now we put all incoming into the file
                output_file(sink_name, data)
        elif sink_type == "odoo":
            odoo_handle(sinks_values, conf, data)
```

Reply:
changed according to your suggestion.